server/src/app.ts

```diff
@@ -15,6 +16,8 @@ const messageError = (err: Error, request: Request, respose: Response, next: Nex
   })
 }
 
+app.use(cors)
```

Comment:
Confirma se não deveria chamar essa função 
```suggestion
app.use(cors())
```

Reply:
https://www.npmjs.com/package/cors